### DISCUSSION TITLE
messed up display

### DISCUSSION THREAD

**Discussion by petriQore** [JanDeDobbeleer/oh-my-posh#5469] [Troubleshoot] [answered]
This is WSL, I used the manual install and I'm already using a nerd font.

![image](https://github.com/user-attachments/assets/feb2e2a1-87fc-497a-b69f-630e6b524e25)

**Comment by shaneholloman** [comment, marked as answer]
yeah, we need to roll back till @lewis-yeung has a fix deployed

If you installed via brew:
```bash
wget https://raw.githubusercontent.com/JanDeDobbeleer/homebrew-oh-my-posh/f6ba331a0509fb5e14fbb0c3d61c5d5b40af2858/oh-my-posh.rb

brew install oh-my-posh.rb
```

**Reply by petriQore** [reply]
Thanks a lot!

**Reply by lewis-yeung** [reply]
Fixed in the latest version.
